src/inc/fixuppointer.h

```diff
@@ -594,4 +594,20 @@ class RelativePointer32
     INT32 m_delta;
 };
 
+#if defined (_TARGET_ARM_) && defined (PLATFORM_UNIX)
```

Comment:
I have mentioned in the issue that it is fine to do this for all platforms. I do not think we need this ifdef.